Pilot/MessageSender.py

```diff
@@ -0,0 +1,266 @@
+""" Message sender module for the remote loggin system.
+    It provides the general interface of the message sender and
+    several implementations that e.g. allows to send the message
+    to a REST interface or to a MQ server using the Stomp protocol.
```

Comment:
Since we agreed at the workshop to provide more technical documentation in the code, can you add here few lines of example on how to use this module?

Reply:
I added some more lines. I hope it is enough